types/index.d.ts

```diff
@@ -26,19 +26,11 @@ declare namespace Realm {
     interface CollectionChangeSet {
         insertions: number[];
         deletions: number[];
-        modifications: number[];
         newModifications: number[];
         oldModifications: number[];
     }
 
-    interface ObjectChanges {
-        insertions: Object[];
```

Comment:
I see these removed. aren't they needed since these are arrays of Objects and the others are arrays of number?

Reply:
Those are the old declarations (now known as `CollectionChangeSet `), I suspect a merge slip has occurred.